doc/nrf/ug_nrf7002.rst

```diff
@@ -0,0 +1,248 @@
+|           |            | ieee802154           |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| QSPI      | on-chip    | qspi                 |
++-----------+------------+----------------------+
+| SPI(M/S)  | on-chip    | spi                  |
++-----------+------------+----------------------+
+| UARTE     | on-chip    | serial               |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+Other hardware features are not supported by the |NCS| kernel.
+
+Connections and IOs
+===================
+
+The following are the connections and IOs supported by the development kit.
+
+LED
+---
+
+* LED1 (green) = P1.06
+* LED2 (green) = P1.07
+
+Push buttons
+------------
+
+* BUTTON1 = SW1 = P1.08
+* BUTTON2 = SW2 = P1.09
+* BOOT = SW5 = boot/reset
+
+Wi-Fi control
+-------------
+
+* BUCKEN = P0.12
+* IOVDD CONTROL = P0.31
+* HOST IRQ = P0.23
+* COEX_REQ = P0.28
+* COEX_STATUS0 = P0.30
+* COEX_STATUS1 = P0.29
+* COEX_GRANT = P0.24
+
+Security components
+===================
+
+The following security components are available:
+
+* Implementation Defined Attribution Unit (`IDAU`_) on the application core.
+
+  The IDAU is implemented with the System Protection Unit and is used to define secure and non-secure memory maps.
+  By default, all of the memory space (Flash, SRAM, and peripheral address space) is defined to be secure accessible only.
+
+* secure boot.
+
+Programming and debugging
+*************************
+
+The nRF5340 application core supports the Armv8-M Security Extension.
+Applications built for the ``nrf7002dk_nrf5340_cpuapp`` board boot by default in the secure state.
+
+The nRF5340 network core does not support the Armv8-M Security Extension.
+nRF5340 IDAU can configure bus accesses by the nRF5340 network core to have the secure attribute set.
+This allows to build and run secure-only applications on the nRF5340 SoC.
+
+Building secure/non-secure |NCS| applications with Arm TrustZone
+=================================================================
+
+Applications on the nRF5340 can contain a secure and a non-secure firmware image for the application core.
+You can build the secure image using either |NCS| or `Trusted Firmware M`_ (TF-M).
+You must always build the non-secure firmware images using |NCS|.
+
+.. note::
+   By default, the secure image for the nRF5340 application core is built using TF-M.
+
+Building the firmware with TF-M
+-------------------------------
+
+To build the secure firmware image using TF-M and the non-secure firmware image using |NCS|, follow these steps:
+
+1. Build the non-secure |NCS| application for the application core using the ``-DBOARD=nrf7002dk_nrf5340_cpuapp_ns`` build target.
```

Comment:
```suggestion
1. Build the non-secure |NCS| application for the application core using the ``nrf7002dk_nrf5340_cpuapp_ns`` build target.
```

Isn't -DBOARD= a CMake parameter to be added at build time in the CLI?
If yes, let's keep this description "neutral" of CLI parameters, as the build target can be set also through VScode : https://nrfconnect.github.io/vscode-nrf-connect/connect/build_app.html